mobile/src/src/components/Community/Feed/FeedContainer.jsx

```diff
@@ -128,13 +128,49 @@ class FeedContainer extends Component {
     return list;
   };
 
-  renderItem({ item }) {
+  attendEvent = (id, shouldAttend) => {
+    let index = this.state.events.findIndex((obj) => obj.id == id);
+    let copyList = this.state.events;
```

Comment:
This doesn't make a copy of each event, it actually refers to the same objects in memory.